pallets/msa/src/mock.rs

```diff
@@ -167,7 +165,7 @@ impl pallet_msa::Config for Test {
 	type MaxProviderNameSize = MaxProviderNameSize;
```

Comment:
Should we consider using MAX_PROVIDER_NAME_SIZE or similar for constants/parameters? (Please let me know if there is a naming convention I haven't read yet.)

Reply:
https://rust-lang.github.io/api-guidelines/naming.html